### PR TITLE
meeting record for 180624(for test)

### PR DESCRIPTION
**Pull request by Tina66**
meeting record for 180624

### CODE REVIEW

**Comment by wangqing1218** [comment]
I think we can use this frame.

**Comment by zmzhouXJTU** [comment]
I agree with WangQing

File: Meeting_Recorde_180624.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  Wang Qing and JinYuxin
+
+Purpose of Meeting:
+  summary and plans for nba-tall and snails project
+Attendance at Meeting:
+  @MrKingHH @hxlong941110 @jsondrm @throcean @wangqing1218 @zmzhouXJTU @Tina66 @July-zhang
+
+Meeting Agenda:
+  Speak freely about the project subject
+Ation Items:
```
Comment (wangqing1218) [inline Meeting_Recorde_180624.md:19]: Action

File: Meeting_Recorde_180624.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+   (1)Create aliyun server(@MrKingHH)
+	 (2)Finish the demos of **DIS** and **CS** of **nba-tall**.(@wangqing1218)
```
Comment (wangqing1218) [inline Meeting_Recorde_180624.md:21]: Fix the format.

**Review by wangqing1218 — APPROVED** [review]

File: Meeting_Recorde_180624.md (hunk 2 above)
Comment (Tina66) [reply Meeting_Recorde_180624.md:21]: Have fixed the format(the final commit).

**Pull request merged**